components/FilterSection/index.js

```diff
@@ -28,6 +28,7 @@ const FilterButton = styled.button`
 `;
 
 const StyledFilterCategorySection = styled.div`
+  font-size: 15px;
   display: flex;
   flex-direction: row;
```

Comment:
```suggestion
```
In almost every case, you can forego `flex-direction: row` because it's the default.